src/js/mock.js

```diff
@@ -0,0 +1,35 @@
+// TODO:
+// - Create our own bundle of faker
+import { faker } from '../vendor/faker.js';
+
+export default function mock() {
+  const response = {
+    "count": 0,
+    "next": null,
+    "previous": null,
+    "results": [],
+  }
+  const limit_results = 20;
+
+  for (let i = 0; i < limit_results; i++) {
+    response.results.push({
+      "answers": [],
+      "created": "2025-10-20T13:28:24.769350Z",
```

Comment:
I believe it's a good option to use the current date for the `created` response field, as it avoids necessary changes to the mock. What do you think?